Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -57,7 +57,7 @@
            10824.521,
            11624.911, 11642.911,
            11601.0,11602.0,11603.0,11604.0,11605.0,11606.0,11607.0,11608.0,11609.0,11630.0,11643.0,
-           11650.0,11634.0,11646.0,11640.0,11834.0,11834.99,11846.0,12024.0,
+           11650.0,11634.0,11646.0,11640.0,11834.0,11834.99,11846.0,12024.0,11925.0,12325.0
```

Comment:
I think a comma (,) is missing after 12325.0

Reply:
Sorry again.